### ISSUE TITLE
Missing dependencies / build instructions

### ISSUE THREAD

**Issue by trib0r3** [GregoryMorse/GhidraDec#1]
Hello, several issues:

1. There is no info about build dependency: `https://github.com/lexxmark/winflexbison`
2. Default _winflexbison_ location should also be mentioned (`%userprofile%\Documents\winflexbison`)
  - files from `./custom_build_rules/win_flex_bison` should be moved one directory up
3. other build issues, cmake is trying to include `decompile/sleigh.cc`, but this file doesn't exist. 

Please test your instruction on the **clean** environment and update instructions :)

**Comment by GregoryMorse** [comment]
Yes I had planned on fixing the cmake environment to have proper downloading of a customizable version of Ghidra sources, right now it hard codes the v1.0 sources that I had modified to fix some bugs.  But doing some diff hot source patches in the cmake process would be far better for that sort of thing.  This means that the Ghidra folder needs to be renamed decompiler.  Anyway the next version will support Ghidra 9.2 and Ida Pro 7.5 with better build process being the primary thing to do.

You are right about the bison dependency needing better documentation obviously on Win vs. Linux it would be a bit different too.

**Comment by GregoryMorse** [comment]
PropertySheet.props contains custom configuration for Visual Studio building, and MakeGhidraDec.bat has the path of CMake and win_flex_bison also required for packaging with this script.

I need to update the readme with these details and I will close the issue.

The decompiler sleigh component is not yet integrated into the CMake script.  I could upload the Ghidra 10 necessary files, but I would prefer to build a CMake script which downloads the correct release, and extracts exactly the needed files.  That is why sleigh.cc the first file causes this issue.

**Comment by GregoryMorse** [comment]
winflexbison is no longer a dependency after the Ghidra 12.1 updated so closing this.
